package.json

```diff
@@ -43,7 +43,7 @@
     "codecov": "1.0.1",
     "commitizen": "2.9.5",
     "cross-env": "3.1.4",
-    "cz-lerna-changelog": "1.0.0",
+    "cz-lerna-changelog": "0.3.2",
```

Comment:
Hmm that's weird. They did a wrong release maybe?

![image](https://cloud.githubusercontent.com/assets/1110551/22101494/fea39942-de33-11e6-964e-4df076cedc33.png)

> Btw. why are we not using the carets?

For dev deps we might, but we shouldn't use that for prod deps. Talking for experience seeing how many "patch", "minor" changes simply break stuff. Also, since we are using `yarn`,  we have a lockfile which locks the deps version.

> Having exact versions like this will require a lot of maintenance.

Not really, using something like e.g. greenkeeper does the maintainance for us.